src/modules/commander/PreflightCheck.cpp

```diff
@@ -152,7 +152,7 @@ static bool magnometerCheck(orb_advert_t *mavlink_log_pub, unsigned instance, bo
 
 	if (ret != OK) {
 		if (report_fail) {
-			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: MAG #%u SELFTEST FAILED", instance);
+			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: MAGNETOMETER #%u SELFTEST FAILED.", instance);
```

Comment:
Capitalization and wording should be updated for consistency with this PR: https://github.com/PX4/Firmware/pull/10586/files
Suggested new text: "Preflight Fail: Compass #%u self test failed."